src/docs/development/ios-14.md

```diff
@@ -28,23 +29,68 @@ Only the debug version of your app needs to access the local network.
 For more information,
 see [Issue 60634]({{site.github}}/flutter/flutter/issues/60634).
 
+## Launching Flutter with Flutter tools
 
-## Launching the app from the home screen
+Due to low-level changes in iOS’s debugger mechanisms,
+developers using versions of Flutter earlier than 1.20.4 stable
+won't be able to launch apps (by using `flutter run`
+or a Flutter-enabled IDE) on physical iOS devices
+running iOS 14. This affects **debug**, **profile**, and
+**release** builds. Simulator builds, add-to-app modules,
+and running directly from Xcode are unaffected. 
 
-When you install a debug version of your app on a physical
-iOS 14 device (using `flutter run` or from Xcode), then disconnect,
-and then try to open the app from the home screen, it crashes.
+Upgrading to Flutter 1.22 beta allows you to build,
+test, and deploy to iOS without issue. Upgrading to
+1.20.4 stable allows you to build and deploy to iOS 14,
+but not debug.
 
-**Solution**: Use `flutter run --release` (or `--profile`)
-if you want to start your app from the home screen.
+## Debugging Flutter
 
-Once again, the crash only happens in debug mode,
-and only when opening the app from the home screen on iOS 14.
-This crash _doesn't_ happen when using the iOS simulator.
+Due to added security around local network permissions in
+iOS 14, a permission dialog box must now be accepted for
+each application in order to enable Flutter debugging
+functionalities such as hot-reload and DevTools. 
 
+![Screenshot of "allow network connections" dialog]({% asset development/device-connect.png @path %})
 
-For more information,
-see [Issue 62531]({{site.github}}/flutter/flutter/issues/62531).
+This affects debug and profile builds only and won't
+appear in release builds. 
+
+For add-to-app users, one additional step has been added
+to the [add-to-app project setup guide][],
+to re-enable flutter attach for debug builds on physical
+devices on iOS 14. 
+
+[add-to-app project setup guide]: /docs/development/add-to-app/ios/project-setup
+
+## Launching debug Flutter without a host computer
+
+Also due to changes in debugger mechanisms, once a Flutter
+debug application is installed on the device
+(either by using `flutter run` a Flutter-enabled IDE,
+or from Xcode), the application can no
+longer be re-launched by tapping the application’s icon
+in the home screen in iOS 14 on physical devices. 
+
+Other launch paths without a host computer, such as deep links
+or notifications, won't work on iOS 14 physical devices in debug mode. 
+
+Add-to-app debug mode modules will crash on iOS 14
+physical devices when running the [`FlutterEngine`][]
+if the host application is launched from the home screen.
+
+To launch the application in debug mode on a physical
+device again, re-run the app (by using `flutter run`,
```

Comment:
re-run the app from the host computer. Just 're-run the app' is ambiguous since that's what the user's trying to do by launching from the home screen.